routes/products.js

```diff
@@ -59,5 +59,18 @@ productsRouter.patch('/:id', isAdmin, async (req, res, next)=>{
         });
     }
 });
+
+productsRouter.delete('/:id', isAdmin, async (req, res, next) => {
+    try {
+        const { id } = req.params;
+        const destroyedProduct = await destoryProduct(req.id);
```

Comment:
this should be `const destroyedProduct = await destoryProduct(id);` instead of `req.id`, since we get the `id` out of the params in the line above.